src/pages/MyTokens/NFTPage.tsx

```diff
@@ -93,11 +94,16 @@ export const NFTPage = () => {
   const [tokens, setTokens] = useState<NFTToken[]>([]);
   const [isFetchingTokens, setIsFetchingTokens] = useState<boolean>(false);
   const deviceSize = useDeviceSize();
+  const navigate = useNavigate();
 
   const { offers, isFetching: isFetchingOffers, fetch } = useOffers();
 
   const { api } = useApi();
 
+  useEffect(() => {
+    if (!isLoading && !selectedAccount) navigate('/');
```

Comment:
1. it looks out of place (redirect from NFT page? Why)
2. Don't redirect any one, no need to. We disabled buttons, that's all, no force.

Reply:
removed